src/openrct2/config/Config.cpp

```diff
@@ -283,6 +284,7 @@ namespace Config
         writer->WriteBoolean("show_real_names_of_guests", model->show_real_names_of_guests);
         writer->WriteBoolean("use_virtual_floor", model->use_virtual_floor);
         writer->WriteBoolean("allow_early_completion", model->allow_early_completion);
+        writer->WriteBoolean("balloons_explode", model->balloons_explode);
```

Comment:
If we wrap a check around this that checks if it's set to true, this option won't be visible in `config.ini` when unset, making it even more of an easter egg.